src/components/ScrollTrack/docs/ScrollTrack.md

```diff
@@ -129,3 +129,30 @@ Example use case: *A container of item cards that all have a width of 90px and m
     <ScrollTrack>
       <CustomComponent />
     </ScrollTrack>
+
+
+Using callbacks
+```js static
+  <ScrollTrack
+    onBeforeNext={(props) => {
+      return new Promise((resolve, reject) => {
+          setTimeout(() => {
+            console.log('before next!', props)
+            resolve()
+          }, 1000)
+      })
+    }}
+    onAfterNext={(props) => { console.log('after next!', props) }}
+    onBeforeBack={(props) => { console.log('before back!', props) }}
+    onAfterBack={(props) => { console.log('after back!', props) }}
```

Comment:
Not sure what use cases these sync callbacks would have, might be good to add an example or call them out specifically.

Reply:
A lot of carousel libraries include them so that carousel actions can trigger or control other aspects of a site/component.